glean-core/csharp/csharp/LibGleanFFI.cs

```diff
@@ -0,0 +1,65 @@
+﻿// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+using System;
+using System.Runtime.InteropServices;
+using System.Text;
+
+namespace Mozilla.Glean.FFI
+{
+    static class LibGleanFFI
```

Comment:
We should add `sealed ` here.

Reply:
Unfortunately, we can't have both `static` and `sealed` on the same class. However, this was clearly lacking the `internal`, so I added it! Good catch!